lib/Github/Api/Organization/Members.php

```diff
@@ -54,8 +54,7 @@ public function conceal($organization, $username)
         return $this->delete('/orgs/'.rawurlencode($organization).'/public_members/'.rawurlencode($username));
     }
 
-    /*
-     * Add user to organization
```

Comment:
As StyleCI points out here, there's a syntax error here.

<img width="1036" alt="image" src="https://user-images.githubusercontent.com/2829600/32809792-3407e410-c990-11e7-89c6-5a28d0608dce.png">

Should be:

```php
    /*
     * Add user to organization
     */
    public function add($organization, $username)
```

Instead of

```php
         * Add user to organization
     */
    public function add($organization, $username)
```

Reply:
Also broken sniff caching "organization" as property name.

Will fix it, thanks